sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -51,13 +51,6 @@ public Meter get(String instrumentationName) {
     return meterBuilder(instrumentationName).build();
   }
 
-  @Override
-  public Meter get(String instrumentationName, String instrumentationVersion) {
-    return meterBuilder(instrumentationName)
-        .setInstrumentationVersion(instrumentationVersion)
-        .build();
-  }
```

Comment:
It looks like you dropped this because the default method in `MeterProvider` covers it. By that logic, I think you can drop `public Meter get(String instrumentationName)` as well.

Reply:
Fixed, good call out!